internal/task/task/bs/format.go

```diff
@@ -188,8 +190,8 @@ func NewFormatChunkfilePoolTask(curveadm *cli.CurveAdm, fc *configure.FormatConf
 	chunkfilePoolRootDir := layout.ChunkfilePoolRootDir
 	formatScript := scripts.FORMAT
 	formatScriptPath := fmt.Sprintf("%s/format.sh", layout.ToolsBinDir)
-	formatCommand := fmt.Sprintf("%s %s %d %d %s %s", formatScriptPath, layout.FormatBinaryPath,
-		usagePercent, DEFAULT_CHUNKFILE_SIZE, layout.ChunkfilePoolDir, layout.ChunkfilePoolMetaPath)
+	formatCommand := fmt.Sprintf("%s %s %d %d %s %s %d", formatScriptPath, layout.FormatBinaryPath,
+		usagePercent, chunkSize, layout.ChunkfilePoolDir, layout.ChunkfilePoolMetaPath, blockSize)
```

Comment:
Now block size exist in chunkserver.conf, mds.conf, curveadm format, hoow to ensure the unity of them?

Reply:
> Now block size exist in chunkserver.conf, mds.conf, curveadm format, hoow to ensure the unity of them?

If chunkfilepool is enabled, blocksize in chunkserver.conf will be overwritten from chunkfilepool's meta which is created by curveadm.

And when deploying the cluster, in the registering process chunkserver will report its blocksize to mds, if blocksize is different between chunkserver and mds, the register will fail.